### DISCUSSION TITLE
multiple "hud groups/layers" that can be changed via a key press

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#20431] [Ideas] [answered]
My idea for this feature is to allow for multiple skin layers that have different elements. For example, I like to play with only hit error/accuracy visible, but not combo/score/timeline. This is good for gameplay, but it is nice to be able to see how I'm actually doing during a break/slow section of a map.

My ideal solution would be allowing multiple "layers" that can be cycled with shift+tab, or simply be able to have a minimalistic layer visible by default, then a more informative layer visible when holding ctrl.

**Comment by integerrr** [comment, marked as answer]
#17978
